salt/modules/solaris_shadow.py

```diff
@@ -190,6 +197,66 @@ def set_mindays(name, mindays):
     return False
 
 
+def gen_password(password, crypt_salt=None, algorithm='sha512'):
+    '''
+    .. versionadded:: 2014.7.0
```

Comment:
The 2015.8.8 release is in testing right now, so 2015.8.9 would be the way to go.